packages/lib-classifier/src/store/SubjectViewerStore/SubjectViewerStore.js

```diff
@@ -82,6 +83,11 @@ const SubjectViewer = types
         self.fullscreen = false
       },
 
+      invertView () {
+        console.log('invert view')
```

Comment:
```suggestion
```

Just checking if this log is intentionally here - I don't think it's necessary for production.

Reply:
I can remove, but included to match resetView, rotate, zoom in, and zoom out, which have similar console.logs, though I'm not entirely sure why. I can remove just invert, remove all, or keep all (as is)?